packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -201,6 +201,11 @@ export interface IDetailsListProps extends React.Props<DetailsList>, IWithViewpo
    * The default implementation will virtualize when this callback is not provided.
    */
   onShouldVirtualize?: (props: IListProps) => boolean;
+
+  /**
+   * Boolean value to indicate if the cells should be vertically centered rather than top-aligned
+   */
+  verticallyCenterCells?: boolean;
```

Comment:
@cschleiden What do you think? This is what we've resolved:

1. The default of vertically align top makes the most sense
2. In some cases, we need to center the content

So the #2 is what @MaxLustig is dealing with and right now it's tricky to do so.

Maybe this should just be done through column definitions. I think column takes a className that gets injected on each cell, could that be a solution here?

Reply:
I'm really, really against this property. We have to be more modular, not less. Can't you do it with an `onRender` callback in the column?

Reply:
I tested out using className on the column definition, which works perfectly well. That'll work for my scenario at least. However, there was a change last week that vertically centers the checkmark in details row, which I had fixed as part of this change. Should I just get rid of all the centering code and keep that change or open up a new PR?

Reply:
Actually, I found one issue with using className in column definition, in that the checkbox does not take a className (and therefore cannot be centered). I've updated the PR to enable providing a css class for the checkbox's cell.